Examples/SubDimensionProcess/Documentation.rst

```diff
@@ -0,0 +1,48 @@
+.. _lbl_sub_dimension_process:
+
+Sub-Dimension Adaptive Histogram Equalization
+=============================================
+
+
+Overview
+--------
+
+Most SimpleITK filter can only operate on 2 or 3 dimensional images, with the exception of filters such as
+`ExtractImageFilter`, `PasteImageFilter`, `SliceImageFilter` and `JoinSeriesImageFilter`. However, SimpleITK
+(by default) supports upto 5 dimensional images. A high dimensional image can be processed by extracting 2 or 3
+dimensional images, then either using the `JoinSeriesImageFilter` to join the sub-volumes together or the
+`PasteImageFilter` to copy the results back to the original image. Possible reasons include when the z direction
+spacing to too great, or for computation or memory efficient reasons. Additionally, it may be desired to process a
+volume (3d or higher) as a sequence of 2 dimensional image.
+
+In this example, the `AdaptiveHistogramEqualizationImageFilter` is used to processes a higher dimensional image as 2
+dimension images. When the filter is run only a single X, Y cross-section of the volume.
+
```

Comment:
...image as 2 dimension images. -> ...image as a set of two dimensional images.

Reply:
done